submissions/Oleksii_Drohachov/OOP-exercise/index.js

```diff
@@ -0,0 +1,76 @@
+class Mammal {
+  constructor(species, name, gender, saying, friends, legs) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+    this.legs = legs;
+    this.propsArray = [
```

Comment:
Data type name as a part of variable name is not necessary (think if you switch to `Map` data type, you'd need to rename variable to reflect the change).
Plural noun is just sufficient.

Also `props` is quite specific term and kinda technical (as in e.g. "component props"). Here we mean "properties of inhabitant", just a normal word that in real life is never abbreviated. Do not save keystrokes on names, IDE will help you to autocomplete even German-language-style variable names.